ocfweb/api/stats.py

```diff
@@ -1,9 +1,15 @@
 from django.http import HttpRequest
 from django.http import JsonResponse
+from ocflib.lab.stats import humanize_bytes
 
+from ocfweb.stats.mirrors import bandwidth_semester
+from ocfweb.stats.summary import desktop_profiles
+from ocfweb.stats.summary import printers
 from ocfweb.stats.summary import staff_in_lab
 from ocfweb.stats.summary import users_in_lab_count
 
```

Comment:
```suggestion
from typing import List, Tuple
```

This is for the suggestion that uses these below.